src/codecTools/createDefaultCodec.ts

```diff
@@ -116,6 +116,11 @@ export const createDefaultCodec = <C extends t.Mixed>(
     return codec.value as t.TypeOf<C>;
   }
 
+  // The default of an object type is an empty object
+  if (codec instanceof t.ObjectType) {
+    return {} as t.TypeOf<C>;
```

Comment:
Would be useful to do this for t.UnknownRecord since t.object is deprecated

Reply:
Pushed another PR here https://github.com/transcend-io/type-utils/pull/35